### DISCUSSION TITLE
would like to see "forgot password" feature

### DISCUSSION THREAD

**Discussion by westbrookc16** [remix-run/remix#3522] [Proposals]
I realize having a forgot password would entail sending an email, but even if we had the message link and ui for resetting passwords and could fill in our email provider. This is a must have feature for those sites that have passwords in my opinion.

**Comment by martijnvandenboom** [comment]
Currently I am working on user registration with an e-mail address. The basics are working but the only thing missing now is the forgot password link and mechanism that will send an e-mail to the user with a link to a page where it will allow the user to actually enter a new password and save it to his or her account. Does someone have a tutorial for that? Or is anyone interested to help me out? My project is in Github maybe we can collaborate?
